torchvision/prototype/datasets/_home.py

```diff
@@ -0,0 +1,20 @@
+import os
+import pathlib
+from typing import Optional, Union
+
+from torch.hub import _get_torch_home
+
+HOME = pathlib.Path(_get_torch_home()) / "datasets" / "vision"
+
+
+def home(home: Optional[Union[str, pathlib.Path]] = None) -> pathlib.Path:
```

Comment:
FYI the function name being equal to the argument could be misleading (recursive function?)

Reply:
Yeah, let's just rename this to `root` or something like that.